AnaProd/SkimEvents.py

```diff
@@ -71,8 +73,12 @@ def make_df(inputFileCentral,inputFileShifted,outDir,treeName,treeName_in='Event
   df_out_valid = df_out.Filter('isValid')
 
   colToSave_diff= []
-  colToNotToMakeDiff = ["period","run", "sample_name", "sample_type", "entryIndex", "event", "isData", "luminosityBlock", "X_mass", "X_spin"]
+  colToNotToMakeDiff = ["period","run", "sample_type", "FullEventId", "event", "isData", "luminosityBlock","X_mass", "X_spin"]
```

Comment:
Why are there so many columns in this list? I think it should contain only `FullEventId`